### ISSUE TITLE
Importing subscriptions "fails" if at least one of the referenced channels has been terminated by Youtube

### ISSUE THREAD

**Issue by syncng** [FreeTubeApp/FreeTube-Vue#129]
Importing subscriptions is this situation results in a infinite loop between local and Invidious API (provided, of course, that said option is enabled)
Here's a screenshot of the supposed "loop"
![Screenshot_2](https://user-images.githubusercontent.com/41585298/92842613-52249680-f3d3-11ea-9e07-e9a21892c71e.png)

**Comment by PrestonN** [comment]
Yep.  That's definitely an issue.  I'll look into this and try to get it fixed.

**Comment by MetalNeo** [comment]
This was only happening for me when using exporting using the Freetube and Newpipe formats specifically. The issue did not occur when using the OPML format.

**Comment by syncng** [comment]
I can confirm that this issue also occurs for me when using the OPML format, I tested all formats.
I'm not sure why that's not the case on your end, you might wanna look into why.

**Comment by PrestonN** [comment]
Are you on the latest update?  I believe I fixed this a couple of days ago.  Please try again and if the issue is fixed then please close this issue.

**Comment by syncng** [comment]
I just tested it on latest.

FreeTube format imports work... well, kind of. The channels that have been terminated simply remain on the subscriptions list along with everything else. When you actually try to navigate to one of those, the toast display doesn't look pretty. Not sure if that's intended behavior since the FreeTube format is much, much different compared to the other two.

As for the remaining two (OPML and NewPipe formats), the only thing that's changed is that the error message of the local API changed to an appropriate termination message, **the loop is still present and there's no way to stop it**.

I doubt you consider that an actual fix.

**Comment by PrestonN** [comment]
Alright.  I've fixed the looping issue.  Please give that another try.

Also with the FreeTube file import, it isn't really a problem with the import itself, it's more so of how I handle a page with an error.  This should be a separate issue.  Also with the FreeTube import, I only use the information within the file whereas the other formats I have to make calls to each of your subscriptions to get the information I need.  That's why with the other formats I can detect that a channel was terminated and can skip it.  These calls are expensive and can risk your IP being blocked by YT if they're done too much, so I'd rather avoid making mass calls whenever I can.  That's why the FreeTube import doesn't do it.

If the importing itself is fixed, then we can close this issue.  The issue of an unpleasant view when viewing a banned channel should be a separate issue from this.

**Comment by syncng** [comment]
The importing is fixed.
It'd be nice to get a file that had the channel names that were unabled to be imported, but that's an afterthought that I'll leave in your hands.
Closing.
